Source/WebKit/WebProcess/WebCoreSupport/WebContextMenuClient.cpp

```diff
@@ -59,7 +59,7 @@ void WebContextMenuClient::searchWithGoogle(const WebCore::Frame* frame)
         return;
 
     auto searchString = frame->editor().selectedText();
-    searchString.stripWhiteSpace();
+    searchString = searchString.stripWhiteSpace();
```

Comment:
I would have put the stripWhitespace() at the end of the previous line.

Reply:
Sure

Reply:
OK, fixed.